### ISSUE TITLE
Fix `studio_spec` system tests

### ISSUE THREAD

**Issue by flotwig** [cypress-io/cypress#18498]
`studio_spec` began failing after #16354.

It is currently skipped in that PR but should be fixed.

https://app.circleci.com/pipelines/github/cypress-io/cypress/24949/workflows/9a3333e5-0255-4bf2-962c-9a9c7ce8d755/jobs/932014

**Comment by flotwig** [comment]
Closing in favor of moving these tests to `app` `e2e` tests in #19263.
